example/README.md

```diff
@@ -37,3 +37,38 @@
    - `./reset-all.sh`
 5. Start the test app
    - `yarn start:example`
+
+## Running (Windows)
+
+1. Open Powershell and navigate to dm-core-packages/example.
+2. Make sure docker engine is running, then run these commands
+   - `yarn install`
+   - `docker-compose down --volumes`
+   - `docker-compose pull`
+   - `docker-compose up -d`
+   - `docker-compose run --rm dmss reset-app` (This fails sometimes, try again if it fails. )
+   - `docker compose run --rm job-api dm -u http://dmss:5000 reset ../app`
```

Comment:
Do you not need to install dm-cli _before_ running this? Would it make sense to add steps to install venv and dm-cli before saying running docker-compose pull and up? 🙂

Reply:
I don't think it matters what you do first of installing dm-cli and running the dc commands. The dc commands run in their own environment, and don't require anything of you computer setup once you have a dc file

Reply:
I just have a memory of not being able to run these commands because I hadn't installed dm-cli and/or installed and acivated venv - but I might be remembering wrong 🙂

Reply:
Do not need to install dm-cli before this.